---
geometry: margin=3cm
title: Plans for Cetline's project
date: \today
author: Jun Ishigohoka
colorlinks: true
linkcolor: blue
urlcolor: blue
numbersections: true
fontsize: 11pt
linestretch: 1

---

\pagenumbering{gobble}





\newpage
\pagenumbering{arabic}
\setcounter{page}{1}

# Introduction



## Background and goals


There has been a long debate in biology whether genes control behaviour.
The first genes controlling behaviour were identified in the fruit fly *Drosophila melanogaster*, focusing on behavioural rhythmicity.
*Drosophila*, just like many other animals, has a circadian clock, which means that they can retain behavioural rhythm even after external cues, such as light-dark cycle, are removed.
What are now known as the "core circadian clock genes" were found in *Drosophila* by screening mutants that show abnormal behavioural rhythm.
Some mutations in the core clock genes are known to cause arrhythmicity in behaviour such as the time of eclosion (hatching of adults from pupae).
Normal flies ("wild type") typically eclose around the dawn, but arrhythmic circadian clock mutants are not as punctual: they can eclose at any time of the day.



In our lab, we work on how the precision of expression of phenotype ("phenotypic variability" and "robustness") evolves.
We are interested in under what condition precise or imprecise genetic control of phenotype (your genes tell "you will be exactly 170cm tall" vs "you can be anywhere between 155 and 175 cm tall and no one can predict exactly how tall you will be") is beneficial.
I develop a computational model to predict when high-variability version of a gene ("high-variability allele") is more beneficial than low-variability version of the gene ("low-variability allele"), under a simple assumption that phenotype is controlled by only one gene.
The next step in my project will be to demonstrate the computer simulation-based prediction in actual experiments, letting high- and low-variability alleles compete.
For this, we need an organism with a gene known to affect phenotypic variability.


In my project, I use the circadian clock mutations affecting the "punctuality" of eclosion time as a model of mutations known to affect phenotypic variability (i.e. punctual = low variability, non-punctual = high variability).
I will make a population of flies with wild type (punctual) and mutant (non-punctual) alleles, replicate the selection condition of the computational model over multiple generations, and follow the frequency of punctual and non-punctual alleles in the population over generations.
If the non-punctual allele increases in frequency over generations, we can say that this allele is more beneficial under the condition that I replicate in the experiment, and we can compare it with the prediction based on the simulation.


For this experiment to match my computational model, the experiment needs to fulfill the assumption that the punctuality is controlled solely by the focal circadian clock gene.
Although the genotype should be almost identical between the punctual and non-punctual lines (because the mutant line was generated by introducing mutations to the wild type flies), they have been separated over decades.
There is a possibility that other mutations have arisen the wild type and mutant that affect the eclosion time in addition to the core clock gene: if this is the case, my experiment does not really match the assumption of my computational model.


In your project, you will study **whether the effect of non-punctual and punctual alleles on the eclosion time depends on the other part of the genome ("genomic background")**.
Most of classic lab flies are inbred, meaning that the two copies of the chromosomes (i.e. maternal and paternal chromosomes) are (almost) identical due to repeated within-family mating.
You will get two inbred lines of flies: wild type (punctual) and mutant (non-punctual).
By crossing these two lines (P0, or parent lines), you will generate F2 flies (filial 2, meaning second-generation hybrid offspring) that have the same genotype at the focal locus as the wild type and mutant lines (i.e. flies with two punctual alleles, or "wild type homozygotes", and flies with two non-punctual alleles, or "mutant homozygotes") yet with diverse combinations of the two parental lines in the genomic background.
You will measure the eclosion rhythm of these flies (P0 and F2), and compare the punctuality of the same genotype between inbred and diverse genomic backgrounds (P0 mutant homozygotes vs F2 mutant homozygotes, and P0 wild type homozygotes vs F2 wild type homozygotes).
If the difference in genomic background between the wild type and mutant lines affects different levels of eclosion punctuality, you will see different distribution of eclosion time between P0 and F2.








## Overview of the project


In your project, you will perform 

1. Crossing (typical "fly work")
    - Cross wild type and mutant (P0) to obtain F1
    - Cross F1 to obtain F2
2. Eclosion phenotyping
    - Record eclosion of individual flies of P0
    - Record eclosion of individual flies of F2
    - Image and data analysis to determine eclosion time of individual flies
3. Genotyping (lab work and a little bioinformatics)
    - Necessary because you cannot tell the genotype of F2
    - Extract DNA from F2 flies with the phenotype data
    - Run PCR to amplify the circadian clock gene
    - Prepare for high-throughput sequencing
    - Analyse the sequencing data to determine the genotype
4. Data analysis
    - Split the phenotype data based on the genotype
    - Visualise and run statistical analysis to compare distribution of eclosion time between P0 and F2 per genotype



# Plans
## Month 0


Mandatory (ordered roughly by importance)

- [ ] Read papers (see [Appendix](#appendix))
- [ ] Practice virgin collection
- [ ] Schedule experiments
- [ ] Egg lay P0
- [ ] Learn how to use eclosion monitors (see [Appendix](#appendix))
- [ ] Get access to the server
- [ ] Install FIJI (see [Appendix](#appendix))
- [ ] Install `R` and `RStudio`



Optional (You will need to learn these later anyways)

- [ ] Learn `R` ([tutorial](https://www.w3schools.com/r/default.asp))
- [ ] Learn `UNIX` command line ([tutorial](https://command-line-tutorial.readthedocs.io/))


## Month 1 (March)


Mandatory

- [ ] Set up and start eclosion recording of P0 (Canton-S and *tim[01]*)
- [ ] Stop eclosion recording of P0 and store samples.
- [ ] Collect virgins of P0 (Canton-S and *tim[01]*)
- [ ] Cross 1 (Canton-S x *tim[01]*) and egg lay
- [ ] Collect virgins of F1
- [ ] Cross 2 (F1 x F1) and egg lay
- [ ] Set up and start eclosion recording of F2
- [ ] Stop eclosion recording of F2 and store samples.





## Month 2 (April)


Mandatory

- [ ] Learn DNA extraction from Lisa (see [Appendix](#appendix))
- [ ] Learn PCR from Lisa (see [Appendix](#appendix))
- [ ] Learn library preparation for amplicon sequencing from Lisa (see [Appendix](#appendix))
- [ ] Schedule experiments
- [ ] DNA extraction
- [ ] Library preparation
- [ ] Send libraries for sequencing



## Month 3 (May)

- [ ] Learn bioinformatics for genotyping using test dataset (see [Appendix](#appendix))
- [ ] Receive sequence data and determine genotypes (see [Appendix](#appendix))
- [ ] Analyse eclosion time for each genotype in P0 and F2




\newpage

# Appendix {#appendix -}
## Literature {-}

PDF files are found in `literature/` folder

### Introduction to working with flies {-}

- [droso4schools](https://droso4schools.wordpress.com/why-fly/)
- Markstein 2019 : Introduction to *Drosophila* work. Chapters 1-5 are must-read.
- [An introduction to *Drosophila melanogaster*](https://depts.washington.edu/cberglab/wordpress/outreach/an-introduction-to-fruit-flies/)


### Eclosion rhythm in flies {-}

- Konopka & Benzer 1971: The first paper in which circadian clock gene mutants were isolated (and gene-mapped) in *Drosophila melanogaster*.
- Ruf et al 2019: Camera-based eclosion monitor
- Horn et al 2019: Competition experiment between wild type and mutant alleles of a circadian clock gene. The result is highly relevant to the motivation of this experiment.
- Kumar et al 2007: Selection experiment on eclosion rhythm in genetically diverse fly population. The design is similar to the main experiment Jun is going to do after your experiment.

### Amplicon sequencing for genotyping {-}

- [Overview â€“ What is amplicon sequencing?](https://eurofinsgenomics.eu/en/eurofins-genomics/material-and-methods/amplicon-sequencing/)
- Wolf et al 2023: The way genotyping is done in this study is same as your project.


## Protocols {-}

Protocols are found in `protocols_pdf/` folder.

- `prep_papae.pdf`: How to prepare pupae for eclosion recording
- `eclosion_monitor.pdf` : How to use eclosion monitors
- `fiji.pdf`: How to install `FIJI` and custom macros
- `image_analysis.pdf`: How to analyse eclosion with `FIJI` and `R`
- `eclosion_detection.pdf`: Identification of eclosion time in `R`
- XYZ: How to extract DNA
- XYZ: How to run PCR
- XYZ: How to prepare library for sequencing
- XYZ: Bioinformatics of amplicon sequencing analysis







